charter.adoc

```diff
@@ -2,6 +2,8 @@
 
 The RISC-V hardware performance monitoring counters (Zihpm) provide support for counting programmable performance events.  Such events can provide insights into software execution behavior, insights that are critical when tuning a profiled workload.  However, no performance events are standardized, which means profiling tools must comprehend a custom set of events specific to each hardware implementation.  This prevents profiling tools from offering general-purpose, event-based analysis capabilities that can be employed regardless of the underlying hardware implementation.
 
-The Performance Events TG will develop an ISA extension that includes a set of standard performance events and metrics (or formulas of events).  For each standard event, the name and the precise hardware behavior associated with it will be specified, though the event selector value associated with the event will be left up to implementations.  For each standard metric, the name and event formula, including the names of the constituent events, will be defined.
+The Performance Events TG will develop a non-ISA extension that includes a set of standard performance events and metrics (or formulas of events). For each standard event, the name and the precise hardware behavior associated with it will be specified, though the event selector value associated with the event will be left up to implementations. For each standard metric, the name, precise description and event formula, including the names of the constituent events, will be defined.
```

Comment:
A nit, but I like the oxford comma.  So I'd add a comma after "precise description"

Reply:
Added